tests/build_/test_build_request.py

```diff
@@ -2826,3 +2826,44 @@ def test_remove_resolve_composes(self, odcs_url, yum_repos, raised):
                 get_plugin(plugins, plugin_type, plugin_name)
         else:
             assert get_plugin(plugins, plugin_type, plugin_name)
+
+    @pytest.mark.parametrize('reactor_config_map', [
+        None,
+        'reactor-config-map',
+    ])
+    def test_set_config_map(self, reactor_config_map):
+        build_request = BuildRequest(INPUTS_PATH)
+        kwargs = {
+            'git_uri': TEST_GIT_URI,
+            'git_ref': TEST_GIT_REF,
+            'user': "john-foo",
+            'component': TEST_COMPONENT,
+            'registry_uri': "example.com",
+            'openshift_uri': "http://openshift/",
+            'builder_openshift_url': "http://openshift/",
+            'build_image': 'fedora:latest',
+            'base_image': 'fedora:latest',
+            'name_label': 'fedora/resultingimage',
+            'registry_api_versions': ['v1'],
+            'reactor_config_map': reactor_config_map,
+        }
+        build_request.set_params(**kwargs)
+        build_json = build_request.render()
+
+        configmapkeyref = {
+            'name': reactor_config_map,
+            'key': 'config.yaml'
+        }
```

Comment:
Since this is not used in the case that `reactor_config_map` is None, this could be moved into the `if reactor_config_map` block.

Reply:
ok